referencemodels/src/test/java/org/openehr/referencemodels/RMComparedWithBmmTest.java

```diff
@@ -108,10 +108,10 @@ public void compareBmmWithRM() {
         knownDifferences.add(new ModelDifference(ModelDifferenceType.CLASS_MISSING_IN_BMM, "", "VERSIONED_FOLDER", null));
 
         //BMM changed VERSION_STATUS to an enum. For now this remains a string until some further major release
-        knownDifferences.add(new ModelDifference(ModelDifferenceType.TYPE_NAME_DIFFERENCE, "",  "ARCHETYPE_HRID", "version_status"));
         knownDifferences.add(new ModelDifference(ModelDifferenceType.CLASS_MISSING_IN_MODEL, "",  "VERSION_STATUS", null));
 
         knownDifferences.add(new ModelDifference(ModelDifferenceType.PROPERTY_MISSING_IN_BMM, "", "ARCHETYPE_HRID", "semantic_id"));
+        knownDifferences.add(new ModelDifference(ModelDifferenceType.PROPERTY_MISSING_IN_BMM, "", "ARCHETYPE_HRID", "version_id"));
```

Comment:
This I can fix. I have to check how exactly, I'll create a commit :)

Reply:
never mind, this should be in the BMM as a computed property anyway since this method is actually in the specification. So, problem of the BMM, not of the implementation!